### PR TITLE
version: added more version information

### LINKED ISSUE #2560: the etcd `--version` flag should print more information
It would greatly help debugging if etcd would print more version info. I think the way the docker project does this is perfect:

```
$ docker version
Client version: 1.4.1
Client API version: 1.16
Go version (client): go1.3.3
Git commit (client): 5bc2ff8
OS/Arch (client): darwin/amd64
```

### PR DESCRIPTION
**Pull request by MSamman**
added more version information output to aid debugging

Fixes #2560

### CODE REVIEW

File: etcdmain/config.go [hunk 1]
```diff
@@ -223,7 +224,11 @@ func (cfg *config) Parse(arguments []string) error {
 	}
 
 	if cfg.printVersion {
-		fmt.Println("etcd version", version.Version)
+		fmt.Printf("etcd Version: %s\n", version.Version)
+		fmt.Printf("etcd Min Cluster Version: %s\n", version.MinClusterVersion)
```
Comment (xiang90) [inline etcdmain/config.go]: hm... we are not ready to expose this. i need more thinking on this one.

File: version/version.go [hunk 2]
```diff
@@ -28,6 +28,9 @@ var (
 	// MinClusterVersion is the min cluster version this etcd binary is compatible with.
 	MinClusterVersion = "2.0.0"
 	Version           = "2.1.0-alpha.0+git"
+
+	// Git SHA Value will be set during build
+	GitSHA = "(None provided)"
```
Comment (xiang90) [inline version/version.go]: "Not provided (use ./build instead of go build)"

Comment (MSamman) [inline version/version.go]: Yea that seems a little more clear. I'll update.

**Comment by xiang90** [comment]
This LGTM. But can you please:

1. give it a try and confirm it works
2. remove the min cluster ver thing for now
3. squash your commits into one

Thanks for your effort!

**Comment by MSamman** [comment]
Yep will do. Stay tuned.

**Comment by xiang90** [comment]
@MSamman Thanks! Merging.

**Comment by MSamman** [comment]
Your welcome!

**Pull request merged**